### ISSUE TITLE
The solutions given in section 5.4 of WCAG 2.1 should be cited, and their implications discussed.

### ISSUE THREAD

**Issue by jasonjgw** [w3c/wcag#993]
Section 5.4 of WCAG 2.1 gives two alternative solutions to the problem of
conformance of Web pages containing third-party content. A conformance claim
can be made based on "best knowledge", or a "Statement of Partial Conformance"
can be issued. Both of these solutions also seem applicable to the problem
raised by conformance of large, complex, or frequently updated Web sites. It
should be noted that this text is unchanged from WCAG 2.0.

Thus, the availability of these solutions in WCAG 2.x directly undermines the
central thesis of the Conformance Challenges document that large or complex
Web sites, and third-party content, constitute significant difficulties for
the WCAG 2 conformance model.

In revising the Conformance Challenges document, the two options given in
section 5.4 of WCAG 2.1 should be cited and discussed. If it is claimed that
these options do not adequately address the Challenges, then this assertion
should be justified. In addition, statements regarding limitations of the WCAG
2 conformance model should be carefully qualified in a manner that
acknowledges the significance of these solutions, as put forward in WCAG
itself.

The two options presented in section 5.4 can also be seen as a baseline
against which to judge any future conformance arrangements in addressing these
issues. Specifically, any future conformance scheme should be at least as
effective (in terms of the accessibility it engenders for people with
disabilities) than expecting content creators to avail themselves of either or
both of the two options in section 5.4.

In discussing section 5.4, I also think it important to disentangle policy
issues from problems inherent in the WCAG conformance model. For example,
there may be policy reasons not to issue a Statement of Partial Conformance,
but one could argue that these considerations would be better addressed in the
policies themselves rather than in WCAG. Another potential policy response
would be for Web site operators to establish contractual arrangements that
hold third parties responsible (i.e., legally liable) for the consequences of
any non-conformance resulting from their contributions.

Thus, I think the central claims made in the Conformance Challenges document
should be revisited in light of the text already present in WCAG 2, and
appropriate adjustments made.

**Comment by peterkorn** [comment]
@jasonjgw - Thank you for pointing this out.  Section 5.4 suggests 2 potential approaches to 3rd party content and conformance:  monitor it and repair or remove problem content and so claim "conformance", or make a partial claim.  I don't believe that either of those options really gets at the challenge of ensuring that this 3rd party content is accessible.  So I propose adding the following new section 3.1 in our 3rd party challenge section:

3.1 Treatment of 3rd party content and Statements of Partial Conformance
WCAG 2.x speaks to 3rd party content and conformance, in the context of a “Statement of Partial Conformance”.  It provides two options for such content – (i) that pages with 3rd party content may make a determination of conformance “based on best knowledge,” for example by monitoring and repairing non-conforming content within 2 business days, or (ii) make a statement of partial conformance if the page could conform if the 3rd party content were removed.
The provision of monitoring and required repair within a 2 business day window doesn’t address the underlying challenge of pages with (3rd party) content that may be updating tens or hundreds of times within that 2 day window.  For large websites <link to glossary definition “Websites with hundreds of thousands of pages or more”> with a significant amount of 3rd party content, the necessity for human involvement in the evaluation of 3rd party content doesn’t scale.  
A statement of partial conformance doesn’t address the underlying challenge of improving the usability of 3rd party content.  At best it might allow a web page/website visitor to be forewarned in advance that they should anticipate encountering inaccessible content.

**Comment by jasonjgw** [comment]
Even if third-party content is added or updated frequently, it may well be possible to crowd-source the human reviews (for example, of captions for video or text alternatives for images). Thus, under some circumstances, a refinement of "best knowledge" may be a reasonable solution - moving beyond the example of the two-day window to the general principle of the proposal. For now, I think it best to acknowledge the issue and to leave it open to further consideration.

A Statement of Partial Conformance is effective from the perspective of a Web site operator who wishes to delineate responsibilities. However, as @peterkorn notes, the message it sends to prospective users with disabilities and to accessibility evaluators is that the third-party contributions may be entirely uncontrolled with respect to WCAG success criteria, even if this is not the case. There seems no reason in principle why the third parties couldn't make Statements of Partial Conformance that are applicable to their contributions. The entity that controls the resources doesn't have a monopoly on asserting partial conformance. Perhaps future Guidelines could make this clearer, and accommodate multiple partial conformance statements made about a Web page or a set of pages - where, as in WCAG 2, a set of pages may be large.

**Comment by sajkaj** [comment]
So, can I close issue 993?

**Comment by jasonjgw** [comment]
Yes, you can close it, subject to any adjustments to the text that you want to make based on my most recent comment.
